ricoconverter/ricoconverter-convert/src/main/resources/xslt_eac/eac2rico-relations.xslt

```diff
@@ -26,6 +26,8 @@
 			<label>Relation hiérarchique</label>
 			<shortcutIfSubjectIsSourceOfRelation>rico:hasOrHadSubordinate</shortcutIfSubjectIsSourceOfRelation>
 			<shortcutIfSubjectIsTargetOfRelation>rico:isOrWasSubordinateTo</shortcutIfSubjectIsTargetOfRelation>
+			<shortcutIfSubjectIsSourceOfRelation_current>rico:hasDirectSubordinate</shortcutIfSubjectIsSourceOfRelation_current>
```

Comment:
Yes, and there should also be something like shortcutIfSubjectIsSourceOfRelation_past with value rico:hadSubordinate (and the second one, with value rico:wasSubordinateTo)